docs/intro.txt

```diff
@@ -95,7 +94,7 @@ to disk:
     IBMProvider.save_account(token=ibm_token)
     QiskitRuntimeService.save_account(channel="ibm_quantum", token=ibm_token)
 
-To see which devices you can access you can use the ``available_devices`` method on the :py:class:`IBMQBackend` or :py:class:`IBMQEmulatorBackend`. Note that it is possible to pass optional ``instance`` and ``provider`` arguments to this method. This allows you to see which devices are accessible through your IBM hub.
```

Comment:
The local emulator was not listed here? Is this functionality available with the local setup?

Reply:
> The local emulator was not listed here? Is this functionality available with the local setup?

No, it's just a class method on `IBMQBackend`.